From a2eff93f300d125cf10626f22b4e10d80b4ac2b7 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 12 Mar 2015 16:30:29 +0800
Subject: [PATCH 0186/1221] MLK-10425-3 ARM: dts: imx7d: enable mega/fast mix
 property

Add Mega/Fast mix off feature in DSM for i.MX7D, if there is
wakeup source in Mega/Fast domain enabled as wakeup event, it
will NOT be off in DSM.

Signed-off-by: Anson Huang <b20788@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 50c0264..e49b4d2 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -566,6 +566,7 @@
 				compatible = "fsl,imx7d-gpc";
 				reg = <0x303a0000 0x10000>;
 				interrupts = <GIC_SPI 87 IRQ_TYPE_LEVEL_HIGH>;
+				fsl,mf-mix-wakeup-irq = <0x4000000 0x0 0x0 0x0>;
 				#power-domain-cells = <1>;
 			};
 		};
-- 
1.7.5.4

